### PR TITLE
feat: integrate addressable

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-7928

### CODE REVIEW

**Review by oktapodia — APPROVED** [review]
LGTM

**Review by tcheee — COMMENTED** [review]
Need to create the flow end-to-end

File: src/components/Widgets/WidgetEvents.tsx [hunk 1]
```diff
@@ -70,6 +70,8 @@ export function WidgetEvents() {
               (el) => el.index === activeTab,
             )[0].variant,
           },
+          enableAddressable: true,
+          isConversion: true,
```
Comment (tcheee) [inline src/components/Widgets/WidgetEvents.tsx:74]: we should remove the `isConversion` here

File: src/components/Widgets/WidgetEvents.tsx
Comment (tcheee) [inline src/components/Widgets/WidgetEvents.tsx:1]: Please add the widgetExpand event in this file as for the featureCard on this one.

File: src/components/FeatureCards/FeatureCards.tsx [hunk 2]
```diff
@@ -23,20 +26,33 @@ export const FeatureCards = () => {
...
+  const { trackEvent } = useUserTracking();
   const { account } = useAccounts();
   const { data: cards, isSuccess } = useStrapi<FeatureCardData>({
     contentType: STRAPI_FEATURE_CARDS,
     queryKey: ['feature-cards'],
   });
 
   useEffect(() => {
-    const handleWidgetExpanded = async (expanded: boolean) =>
+    const handleWidgetExpanded = async (expanded: boolean) => {
```
Comment (tcheee) [inline src/components/FeatureCards/FeatureCards.tsx:37]: I would create a hook for this useEffect is possible

**Review by tcheee — APPROVED** [review]

**Pull request merged**